cli/cmd/compliance_aws.go

```diff
@@ -216,6 +235,24 @@ func init() {
 	complianceAwsGetReportCmd.Flags().StringVar(&compCmdState.Type, "type", "CIS",
 		"report type to display, supported types: CIS, NIST_800-53_Rev4, ISO_2700, HIPAA, SOC, or PCI",
 	)
+
+	complianceAwsGetReportCmd.Flags().StringVar(&compCmdState.Category, "category", "",
+		"filter the compliance report details view by category (identity-and-access-management, s3, logging...)",
+	)
+
+	complianceAwsGetReportCmd.Flags().StringVar(&compCmdState.Service, "service", "",
+		"filter the compliance report details view by service (aws:s3, aws:iam, aws:cloudtrail ...)",
```

Comment:
Cloud we make this string smaller and consistent? I think this is perfect for all new options:
```suggestion
		"filter report details by service (aws:s3, aws:iam, aws:cloudtrail ...)",
```

Reply:
Done